bin/pasta_prepare_evaluation.py

```diff
@@ -15,15 +15,21 @@
 
 import argparse
 import csv
+import os
 import re
+import pickle
 
 from logging import getLogger
+
+from multiprocessing import Pool, cpu_count
 from subprocess import call
 
-from pypasta.LinuxMaintainers import load_maintainers
+from tqdm import tqdm
+
+from pypasta.LinuxMaintainers import LinuxMaintainers
 from pypasta.LinuxMailCharacteristics import load_linux_mail_characteristics
 
-log = getLogger(__name__[-16:])
+log = getLogger(__name__[-18:])
```

Comment:
Is there a particular reason for this change?

Reply:
The log info doesn't print the whole name of the function otherwise. Length of 'prepare_evaluation' is 18 here. I do not like this format of logging in general, but I just complied with the rest of the logger code.

Reply:
Heh, no worries, I don't like it either! :-)
Need to reconsider the whole logging structure one day...